docs/how-to/uc-install-real-time-kernel.md

```diff
@@ -16,30 +16,25 @@ snap info pc-kernel
 For the list of kernel snaps for other architectures, refer to the [pc-kernel](https://snapcraft.io/pc-kernel) Snap Store page.
 ```
 
-Under `channels`, look for entries that contain `-rt`.
+Under `channels`, look for entries that contain `<version>-rt` as prefix. 
+Each [channel][channels] consists of a track followed by the risk level.
+In general, look at the `stable` risk level to find the most reliable revisions.
 
-The results can be filtered with `grep`: 
+Filter results with `grep`. For example, here are the real-time Ubuntu Core 24 kernels that are stable:
```

Comment:
This should be "Real-time Ubuntu Core [...]" if we are referring to the product, or "real-time kernels available/compatible on Ubuntu Core 24", when referring to the kernels being used.